apps/src/aichat/views/presentation/PresentationView.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ModelCardRow from './ModelCardRow';
+import {MODEL_CARD_FIELDS_AND_LABELS} from '@cdo/apps/aichat/views/modelCustomization/constants';
+import styles from '@cdo/apps/aichat/views/model-customization-workspace.module.scss';
+import {Heading4} from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './presentation-view.module.scss';
+
+const PresentationView: React.FunctionComponent = () => {
+  const modelCardIcons = [
```

Comment:
Could we add these to MODEL_CARD_FIELDS_AND_LABELS (or generate a new constant or something that includes the icon name)? Seems a little fragile to rely on the ordering

Reply:
I thought maybe we could add `examplePromptAndIcons` to `MODEL_CARD_FIELDS_AND_LABELS` eventually since it will be added in edit mode. But I see that it would interfere with @sanchitmalhotra126 's work with the level edit page.